easybuild/easyconfigs/g/genesis/genesis-0.29.0-GCC-11.3.0.eb

```diff
@@ -0,0 +1,40 @@
+easyblock = 'CMakeMake'
+
+name = 'genesis'
+version = '0.29.0'
+
+homepage = 'https://github.com/lczech/genesis'
+description = """Genesis is a C++ library for working with phylogenetic and population genetic data"""
+
+toolchain = {'name': 'GCC', 'version': '11.3.0'}
+toolchainopts = {'pic': True}
+
+github_account = 'lczech'
+source_urls = [GITHUB_SOURCE]
+sources = ['v%(version)s.tar.gz']
+checksums = ['63e7b8c45aeb72cb9503b0b0f5ae72d51715a42bf8ef970ae0ab1ec5dbd26e79']
+
+builddependencies = [
+    ('pkgconf', '1.8.0'),
+    ('CMake', '3.24.3'),
+    ('Autoconf', '2.71'),
+]
+
+dependencies = [
+    ('bzip2', '1.0.8'),
+    ('XZ', '5.2.5'),
+    ('HTSlib', '1.15.1'),
+    ('libdeflate', '1.10'),
+]
+
+skipsteps = ['install']
+
+postinstallcmds = ['mkdir -p %%(installdir)s/lib/ && '
```

Comment:
I was checking our Generic EasyBlocks. It's funny: we have something for software without `configure`, with `make`, and without `make install` (`makeCp`), as well as for software without `configure`, `make` and `make install` (`cmdCp`), but _not_ for software _with_ a `configure`, a `make`, but no `make install`.

We _should_ really have a `ConfigureMakeCp`... But don't worry, I won't force you to make one (but if you feel like it, I'd encourage it ;-))